nomos-services/membership/src/backends/mod.rs

```diff
@@ -0,0 +1,40 @@
+use std::collections::HashMap;
+
+use async_trait::async_trait;
+use nomos_sdp::backends::FinalizedBlockEvent;
```

Comment:
Would it make sense to move this to `services/nomos-sdp/lib.rs`, or even `nomos-sdp-core`, that way membership would require spd-service dependency only in the sdp adapter, and we wouldn't need sdp service for the mock implementation. I'm not sure about this, but it seems that `FinalizedBlockEvent` fits into `nomos-sdp-core`.